dnf-behave-tests/common/output.py

```diff
@@ -80,7 +80,7 @@ def then_stdout_is(context):
     if found == [""]:
         found = []
 
-    dnf5_mode = hasattr(context, "dnf") and context.dnf.dnf5_mode
+    dnf5_mode = hasattr(context, "dnf") and context.dnf5_mode
```

Comment:
But then the `hasattr(context, "dnf")` check is redundant (I wonder why it was needed...) and you can use `context.dnf5_mode` directly instead of this variable?

Reply:
Oh. These are common steps, It must have been because of the createrepo_c tests, which don't have neither `context.dnf` not `context.dnf5_mode`, so I suspect you've broken createrepo_c tests with this?

Reply:
oh, right. We probably need something like `hasattr(contect, "dnf5_mode") and context.dnf5_mode`.
Thanks for pointing this out.

Reply:
Yeah, well I've just realized it shouldn't be broken, the `hasattr(context, "dnf")` should actually do the job, though it's not awfully nice or intuitive :slightly_smiling_face:

Reply:
yes, the createrepo_c CI passed, but it is not nice.
I made a follow up PR: https://github.com/rpm-software-management/ci-dnf-stack/pull/1123 
Thanks!